### ISSUE TITLE
_three undefined 

### ISSUE THREAD

**Issue by TheMaverickProgrammer** [Irrelon/ige#438]
igeThreeComponent lacks documentation.

I ran the webgl multiplayer project and found that the turret did not move. Further inspection revealed an error in javascript code "ige._three" undefined.

**Comment by Irrelon** [comment]
You are correct, the three.js integration is experimental and probably needs some love. If you have time and the inclination it might be a really nice pull request to get it working again! :)

**Comment by TheMaverickProgrammer** [comment]
I don't but I'm sure if I do in the future I'll revisit ige for 3D purposes.
